frontend/package.json

```diff
@@ -54,7 +54,6 @@
     "redux-thunk": "^2.3.0",
     "regenerator-runtime": "^0.13.3",
     "reselect": "^4.0.0",
-    "save": "^2.4.0",
```

Comment:
If a dependency is being removed the package-lock.json file needs to be updated as well. Why is this being removed?

Reply:
This was accidentally added, Likely as a result of forgetting `--` when saving other packages. 

I attempted to remove it in the previous sprint, but with merge conflicts, it kept getting added back in. 

Will remove from the package-lock.json, although I was under the impression that once you ran 'npm ci', the `package-lock.json` would be updated according to the current `package.jon`, is this not the case?